blog/2020/08/invalidations.md

```diff
@@ -330,7 +330,7 @@ It's worth noting that the task of improving code is never finished, and this is
 ## Tools for analyzing and fixing invalidations
 
 Recently, the [SnoopCompile] package gained the ability to analyze invalidations and help developers fix them.
-Because these tools will likely change over time, this blog post will only scratch the surface; people who want to help fix invalidations are encouraged to read [SnoopCompile's documentation](https://timholy.github.io/SnoopCompile.jl/stable/snoopr/) for further detail.
+Because these tools will likely change over time, this blog post will only scratch the surface; people who want to help fix invalidations are encouraged to read [SnoopCompile's documentation](https://timholy.github.io/SnoopCompile.jl/dev/) for further detail.
```

Comment:
Seems like it would be better to link to the `stable` docs instead of the `dev` docs, right?

The following URL seems to work:

```suggestion
Because these tools will likely change over time, this blog post will only scratch the surface; people who want to help fix invalidations are encouraged to read [SnoopCompile's documentation](https://timholy.github.io/SnoopCompile.jl/stable/) for further detail.
```

Reply:
the main page doesn't link to stable docs, which made me think Tim wants people to go to dev docs: https://github.com/timholy/SnoopCompile.jl

Reply:
Hmmm, interesting.

@timholy Which one (`stable` or `dev`) should we link to on the Julia website?

Reply:
I'll merge this to fix the 404. Necessary changes can follow.